### ISSUE TITLE
Java: Error message ”Could not find method testCompile() for arguments [junit:junit:4.13] ” with commanding "gradle test"

### ISSUE THREAD

**Issue by miku0129** [exercism/website-copy#1971]
Hi! I'm new to both Exercism and Java! 
I could succeed with installing Java, gradle, and setting up CLI for Exercism.  And I installed the first project: hello-world. 
When I command `gradle test` at `$ cd /Users/johndoe/exercism/java/hello-world` , the error message says 

`FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\svart\Exercism\java\hello-world\build.gradle' line: 14

* What went wrong:
A problem occurred evaluating root project 'hello-world'.
> Could not find method testCompile() for arguments [junit:junit:4.13] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler. `

I have no idea how to solve this issue. I've never touched build.gradle nor other files. I appreciate any advice.
Thank you!!

**Comment by miku0129** [comment]
I found that gradle 7 released and "testCompile" is no longer supported. 
[](https://tomgregory.com/gradle-implementation-vs-compile-dependencies/)
So there is a possibility that I downloaded the latest version of gradle 7  and the tutorial code is still for gradle 6.

**Comment by angelikatyborska** [comment]
It seems to be the same issue as https://github.com/exercism/java/issues/1944 and https://github.com/exercism/exercism/issues/5517, both have responses.

TL;DR: it was fixed by Java maintainers but it is not yet live on exercism.io due to a track synchronization pause until v3 launches.

I'll close this one.
